docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-logout.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: mesheryctl system logout
+permalink: reference/mesheryctl/system/logout
+type: reference
+display-title: "false"
+language: en
+lang: en
```

Comment:
Why two languages? // @jesuslerma

Reply:
I don't see any particular reason, we don't have a Spanish version of this.

Reply:
I thought I removed them! 😳😳 Oh damn, missed some then :')

Reply:
@alphaX86 to we just required the first language field or no language at all? I'll do the changes in this PR.

Reply:
The ideal one would be only one language field and not two. Maybe you could check

Reply:
Alright will do